src/OpenTK.Platform/Native/Windows/Win32.cs

```diff
@@ -1471,6 +1471,13 @@ internal struct RAWINPUTDEVICE
         [DllImport("user32.dll")]
         internal static extern IntPtr /* LRESULT */ DefRawInputProc(IntPtr /* PRAWINPUT* */ paRawInput, int nInput, uint cbSizeHeader);
 
+        [DllImport("User32.dll", SetLastError = true)]
+        public static extern uint GetRawInputBuffer(
+            [Out, Optional] IntPtr pData,  // PRAWINPUT
```

Comment:
Follow the format of all the other functions. The actual type of the IntPtr should be between the type and the variable name. For this function it doesn't make sense to use `IntPtr` when `RAWINPUT*` is the more correct signature.

Don't use the `[Out]` and `[Optional]` attribute.